module-1/fibonacci.js

```diff
@@ -15,6 +15,20 @@ function fibonacci(n) {
      * Also take into consideration the documentation of the function!
      */
     // PLACE YOUR CODE BETWEEN THIS...
+    
+nThFibonacci = 0;
+let firstSum = 0;
+let secondSum = 1;
+if (n >= 0) {
```

Comment:
It is good practice to make a check at the beginning to see if we have valid values, before even initiating the variables.

Reply:
I agree. I correct this line.